CONTRIBUTING.rst

```diff
@@ -183,7 +183,8 @@ their individual contributions.
 * `kbara <https://www.github.com/kbara>`_
 * `Kyle Reeve <https://www.github.com/kreeve>`_ (krzw92@gmail.com)
 * `Lee Begg <https://www.github.com/llnz2>`_
-* `Louis Taylor <https://github.com/kragniz>`_
+* `Louis Taylor <https://github.com/kragniz>`
+* `Luke Barone-Adesi <https://github.com/baluke>`
```

Comment:
You need an underscore after the closing backtick on these lines for the RST hyperlink syntax.